packages/backend-core/src/sql/sql.ts

```diff
@@ -325,14 +325,18 @@ class InternalBuilder {
     return input
   }
 
-  private parseBody(body: any) {
-    for (let [key, value] of Object.entries(body)) {
-      const { column } = this.splitter.run(key)
-      const schema = this.table.schema[column]
-      if (!schema) {
-        continue
+  private parseBody(body: Record<string, any>) {
+    try {
+      for (let [key, value] of Object.entries(body)) {
+        const { column } = this.splitter.run(key)
+        const schema = this.table.schema[column]
+        if (!schema) {
+          continue
+        }
+        body[key] = this.parse(value, schema)
       }
-      body[key] = this.parse(value, schema)
+    } catch (err) {
+      console.log(err)
```

Comment:
Is it correct to just return the body as-is, un-parsed, when anything goes wrong in parsing? Feels like something that's destined to cause future bugs.

Reply:
Oh woops - I should have removed this try-catch, was just during my debugging, nice catch!